setup.py

```diff
@@ -23,7 +23,7 @@ def package_data(pkg, roots):
 
 setup(
     name='completable-html-xblock',
-    version='0.1.2',
+    version='0.2.0',
```

Comment:
This is a breaking change, so we should bump MAJOR. [Ref](https://semver.org/).

Reply:
@taniwha, ^